Alicerce/Sources/Logging/Log+ItemJSONFormatter.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ItemJSONFormatter.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ItemJSONFormatter: LogItemFormatter {
+
+        public func format(logItem: Log.Item) -> String {
+
+            let dict: [String: Any] = [
+                "timestamp": Date().timeIntervalSince1970,
+                "level": logItem.level.rawValue,
+                "message": logItem.message,
+                "thread": logItem.thread,
+                "file": logItem.file,
+                "function": logItem.function,
+                "line": logItem.line
+            ]
+
+            guard let jsonData = try? JSONSerialization.data(withJSONObject: dict, options: []) else {
+                assertionFailure("failed to convert log item dictionay into JSON data object");
+                return ""
```

Comment:
Hum I understand your point of view here. The thing is, that if you really want to bubble this error up to some level, you must level it up to the code that actually calls one of the ```Log.Info``` and siblings, meaning that you'll have some sort of error handling in these calls, which I really really want to avoid.
The ```assertionFailure``` provides to me a balance between warning the developer that something went wrong during development, and avoiding bubbling up error handling to the client. But also, I have mixed feelings, but I guess I prefer this approach.